tests/test_georaster.py

```diff
@@ -750,28 +771,42 @@ def test_crop(self, data: list[str]) -> None:
             cropGeom[2] - rand_float * r.res[0],
             cropGeom[3] - rand_float * abs(r.res[1]),
         ]
-        r_cropped = r.crop(cropGeom2, inplace=False, mode="match_extent")
+        with warnings.catch_warnings():
+            warnings.filterwarnings("ignore", category=UserWarning)
+            r_cropped = r.crop(cropGeom2, inplace=False, mode="match_extent")
         assert list(r_cropped.bounds) == cropGeom2
         # The change in resolution should be less than what would occur with +/- 1 pixel
         assert np.all(
             abs(np.array(r.res) - np.array(r_cropped.res)) < np.array(r.res) / np.array(r_cropped.shape)[::-1]
         )
 
-        r_cropped2 = r.crop(r_cropped, inplace=False, mode="match_extent")
+        with warnings.catch_warnings():
+            warnings.filterwarnings("ignore", category=UserWarning)
+            r_cropped2 = r.crop(r_cropped, inplace=False, mode="match_extent")
         assert r_cropped2 == r_cropped
 
         # -- Test with CropGeom being a Vector -- #
         outlines = gu.Vector(outlines_path)
-        outlines.ds = outlines.ds.to_crs(r.crs)
-        r_cropped = r.crop(outlines, inplace=False)
+
+        # First, we reproject manually the outline
+        outlines_reproj = gu.Vector(outlines.ds.to_crs(r.crs))
```

Comment:
When reading your PR, I was thinking that in the same effort (making crop be similar between Raster and Vector), we should make reproject more consistent between the two, i.e. have a Vector.reproject function.

Reply:
I was thinking the same, but then reproject would only be about the CRS, right?

Reply:
Yes that's right, but still it would be more consistent between the two classes.

Reply:
Just added issue #340